parlai/chat_service/services/terminal_chat/README.md

```diff
@@ -0,0 +1,18 @@
+## Terminal Chat
+This allows you to participate in a ParlAI world as an agent using the terminal.
+This extends the `websocket` chat service implementation to run a server locally,
+which you can send and receive messages from using the terminal.
+
+## Setup
+1. Run:
+
+ `python parlai/chat_service/services/terminal_chat/run.py --config-path path/to/config.yml --port PORT_NUMBER`
```

Comment:
maybe server_run.py instead?

What is the config yaml file?

Reply:
this is the new way of running these services - it's where all the info is specified (rather than having a separate `run.py` file for each task). It's generally been the convention that `run.py` starts the server

Reply:
that's fine; where is an example config file? It's not provided in the documentation here.

why did we go with yaml? Now we've got json and yaml and pickle?

Reply:
`yaml` is something we had been using internally, there is no "reason" specifically we're using yaml, it looks kinda nice 😄 

Also there is an example config in line 13 of this file

Reply:
alright good enough for me